tokens/connect/ledger.md

```diff
@@ -75,7 +81,7 @@ If a pop-up doesn't appear, you may need to change your MetaMask settings to ena
 !!! note
     The **Preferred Ledger Connection Type** setting is only available on Chrome and Chrome-based browsers. This setting doesn't exist on other browsers such as Firefox.
 
-If MetaMask was able to connect successfully to your Ledger device, you should see a list of five Moonbeam/Ethereum-styled accounts. If not, double-check that Ledger Live is closed, you've connected your Ledger device to the computer, unlocked it, and that the Moonriver app is opened. If you're adapting these instructions for another Moonbeam-based network aside from Moonriver, make sure the Ethereum app is opened on your Ledger device. 
+If MetaMask was able to connect successfully to your Ledger device, you should see a list of five Moonbeam/Ethereum-styled accounts. If not, double-check that Ledger Live is closed, you've connected your Ledger device to the computer, and unlocked it. For Moonbeam and Moonbase Alpha, make sure that the Ethereum app is opened. For Moonriver, make sure that the Moonriver app is opened.
```

Comment:
I wouid split at the bottom between the Ethereum ledger app or the dedicated Moonriver app. In the future, we can just say "dedicated Moonbeam/Moonriver Ledger App"